### PR TITLE
Small fixes and updates for the Portuguese translation

### PR DESCRIPTION
**Pull request by gjvnq**

### CODE REVIEW

File: public/pt/historia.md [hunk 1]
```diff
@@ -55,30 +55,30 @@ Pessoas trans estão saindo do armário; nós estamos em todos os lugares.
...
 
 > Você já estava sentado em algum lugar público ou formal e do nada a sola do seu sapato coça? Você não pode remover seus sapatos ali e coçar o seu pé, então você aguenta a sensação morrendo dentro de você enquanto essa coceira cresce e cresce até que você está pronto para matar a próxima pessoa que falar com você.
 >
 > Ou quando eu era mais jovem e costumava assistir TV à cabo nas manhãs antes da escola. Como era TV à cabo na parte rural da Virgínia Ocidental (estado dos EUA) no começo dos anos 90, de tempos em tempos eu ligava o meu canal favorito e assistiria meus shows enquanto eu comia *maple oatmeal* de café da manhã e assistia Power Rangers — mas o áudio vinha de outra estação (geralmente do canal de tempo meterológico). O vídeo estava okay. O áudio estava okay. Mas o desencontro entre eles? Esse era o tipo de frustração que ficava com você o dia inteiro quando criança.
 >
-> É a sensação de que você tem quando você pede uma refrescante Coca Diet e o garçom responde: “Pode ser Pepsi?”
+> É a sensação de que você tem quando você pede uma refrescante Coca Diet e o garçom responde: “Pode ser ~~Guaraná Jesus~~ Pepsi?”
```
Comment (Twipped) [inline public/pt/historia.md:68]: This was a test, wasn't it? 😏

**Comment by gjvnq** [comment]
It was a joke for the reader.

Guaraná Jesus is/was very popular in the Northeast of Brazil. Also, the
Guaraná Jesus can/bottle design kinda resembles the transgender flag.

https://pt.wikipedia.org/wiki/Guaran%C3%A1_Jesus

**Review by Twipped — CHANGES_REQUESTED** [review]
Everything else looks good, just that one bit of sacrilege needs to be fixed 😏

**Comment by Twipped** [comment]
@gjvnq ah, okay, that context is useful. Thanks!

**Pull request merged**